### PR TITLE
Update OWNERS file

### PR DESCRIPTION
**Pull request by enriquebelarte**
Add user @enriquebelarte to OWNERS.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -6,6 +6,7 @@ approvers:
   - qbarrand
   - yevgeny-shnaidman
   - ybettan
+  - enriquebelarte
```
Comment (qbarrand) [inline OWNERS:9]: Could you please follow the alphabetical order here? Same for the `reviewers` section below. Thanks!

**Comment by ybettan** [comment]
/retest
/hold
Until @qbarrand comment is fixed.

File: OWNERS (hunk 1 above)
Comment (enriquebelarte) [reply OWNERS]: Done

**Review by bthurber — APPROVED** [review]
/lgtm

**Pull request merged**